app/build.gradle

```diff
@@ -125,14 +130,14 @@ task jacocoTestReport(type: JacocoReport, dependsOn: ['testDebugUnitTest', 'crea
     def debugTree = fileTree(dir: "$project.buildDir/intermediates/javac/debug/classes", excludes: fileFilter)
     def mainSrc = "$project.projectDir/src/main/java"
 
-    sourceDirectories = files([mainSrc])
-    classDirectories = files([debugTree])
-    executionData = fileTree(dir: project.buildDir, includes: [
+    sourceDirectories.from = files([mainSrc])
+    classDirectories.from = files([debugTree])
+    executionData.from = fileTree(dir: project.buildDir, includes: [
```

Comment:
?

Reply:
En cherchant le bug de code climate, j'ai decidé de nettoyer des warnings comme "The JacocoReportBase.setSourceDirectories(FileCollection) method has been deprecated. This is scheduled to be removed in Gradle 6.0. Use getSourceDirectories().from(...)".

Cela n'affecte pas le fonctionnement de l'app, c'est juste un leger nettoyage